sympy/geometry/polygon.py

```diff
@@ -402,6 +403,54 @@ def centroid(self):
             cy += v*(y1 + y2)
         return Point(simplify(A*cx), simplify(A*cy))
 
+    def second_moment_of_area(self,p=None):
```

Comment:
There are many pep8 related issues. Please review pep8 and/or use a linter.

Reply:
done